app/src/main/java/edu/grinnell/appdev/grinnelldirectory/activities/SimpleSearchActivity.java

```diff
@@ -1,25 +1,37 @@
 package edu.grinnell.appdev.grinnelldirectory.activities;
 
+import android.content.Intent;
+import android.support.v7.app.AlertDialog;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
 import android.view.View;
 import android.widget.Button;
 import android.widget.EditText;
+import butterknife.BindString;
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
+import edu.grinnell.appdev.grinnelldirectory.DBAPICaller;
+import edu.grinnell.appdev.grinnelldirectory.interfaces.APICallerInterface;
+import edu.grinnell.appdev.grinnelldirectory.models.Person;
+import edu.grinnell.appdev.grinnelldirectory.models.SimpleResult;
+import edu.grinnell.appdev.grinnelldirectory.models.User;
 import edu.grinnell.appdev.grinnelldirectory.R;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * SimpleSearchActivity allows the user to search with one text field.
  */
 
-public class SimpleSearchActivity extends AppCompatActivity {
+public class SimpleSearchActivity extends AppCompatActivity implements APICallerInterface {
     @BindView(R.id.first_name_field) EditText mFirstNameEditText;
     @BindView(R.id.last_name_field) EditText mLastNameEditText;
     @BindView(R.id.search) Button mSearchButton;
 
+    public static final String BUNDLE_KEY = "BUNDLE_KEY";
```

Comment:
This should be unique to the `SimpleSearchActivity` in case someone else wants to have a bundle key.